mods/cnc/maps/nod04a/rules.yaml

```diff
@@ -5,7 +5,7 @@ World:
 		StartingMusic: warfare
 		VictoryMusic: nod_win1
 	MissionData:
-		Briefing: GDI is attempting to relocate a village of sympathetic civilians. Intercept the convoy and destroy it. It is imperative that the villagers be totally destroyed as a lesson to any other natives who may oppose us.
+		Briefing: GDI is trying to relocate a village of sympathetic civilians. Intercept the convoy and destroy it. Ensure that no villagers survive — this will serve as a warning to any who dare oppose us.
```

Comment:
The "other natives" is important flavour here - showing the contempt that Nod has for the local populace.